src/tools/genpolicy/samples/erofs-tarfs-example.rego

```diff
@@ -0,0 +1,1784 @@
+              "rprivate",
+              "rw"
+            ]
+          },
+          {
+            "destination": "/var/run/secrets/kubernetes.io/serviceaccount",
+            "source": "$(sfprefix)serviceaccount$",
+            "type_": "bind",
+            "options": [
+              "rbind",
+              "rprivate",
+              "ro"
+            ]
+          },
+          {
+            "destination": "/var/run/secrets/azure/tokens",
+            "source": "$(sfprefix)tokens$",
+            "type_": "bind",
+            "options": [
+              "rbind",
+              "rprivate",
+              "ro"
+            ]
+          }
+        ],
+        "Annotations": {
+          "io.katacontainers.pkg.oci.bundle_path": "/run/containerd/io.containerd.runtime.v2.task/k8s.io/$(bundle-id)",
+          "io.katacontainers.pkg.oci.container_type": "pod_container",
+          "io.kubernetes.cri.container-name": "nginx",
+          "io.kubernetes.cri.container-type": "container",
+          "io.kubernetes.cri.image-name": "quay.io/chengyu_zhu/nginx:latest",
+          "io.kubernetes.cri.sandbox-id": "^[a-z0-9]{64}$",
+          "io.kubernetes.cri.sandbox-name": "nginx-cc",
+          "io.kubernetes.cri.sandbox-namespace": "default"
+        },
+        "Linux": {
+          "Namespaces": [
+            {
+              "Type": "ipc",
+              "Path": ""
+            },
+            {
+              "Type": "uts",
+              "Path": ""
+            },
+            {
+              "Type": "mount",
+              "Path": ""
+            }
+          ],
+          "MaskedPaths": [
+            "/proc/acpi",
+            "/proc/kcore",
+            "/proc/keys",
+            "/proc/latency_stats",
+            "/proc/timer_list",
+            "/proc/timer_stats",
+            "/proc/sched_debug",
+            "/proc/scsi",
+            "/sys/firmware"
+          ],
+          "ReadonlyPaths": [
+            "/proc/asound",
+            "/proc/bus",
+            "/proc/fs",
+            "/proc/irq",
+            "/proc/sys",
+            "/proc/sysrq-trigger"
+          ]
+        }
+      },
+      "storages": [
+      {
+      "driver": "dmverity",
+      "driver_options": [
+        "verity_info={\"hashtype\":\"sha256\",\"hash\":\"8d564cbd4fd32ded03fedd077db4b26b8e4c07a75dd15f936e58f39abd8f6562\",\"blocknum\":1038,\"blocksize\":512,\"hashsize\":4096,\"offset\":532480}",
+        "source_type=virtio_pci"
+      ],
+      "source": "02/02",
+      "fstype": "erofs",
+      "options": ["ro", "ro"],
+      "mount_point": "/run/kata-containers/virtual-volumes/verity/8d564cbd4fd32ded03fedd077db4b26b8e4c07a75dd15f936e58f39abd8f6562",
+      "fs_group": null
+    },
+    {
+      "driver": "dmverity",
+      "driver_options": [
+        "verity_info={\"hashtype\":\"sha256\",\"hash\":\"e4fe7b12607c54e597741bbedace4c44816f24db23fae98c73c8b7c23c4bb981\",\"blocknum\":1031,\"blocksize\":512,\"hashsize\":4096,\"offset\":528384}",
```

Comment:
e4fe7b12607c54e597741bbedace4c44816f24db23fae98c73c8b7c23c4bb981 gets repeated too. So, I think we need to keep the more complex but smaller data format that you are removing from rules.rego.

What is the size of your policy generated for the YAML file from https://github.com/confidential-containers/confidential-containers/issues/197?

Reply:
I’ll give it a try and share my thoughts with you.

Reply:
Hi @danmihai1. I have tried to create a policy file based on the YAML file that you provided, and the encoded size is 118417 bytes.

Reply:
I found that the size limit for the k8s annotation is 256KB, and I believe it's big enough for policy. WDYT?

Reply:
I tested that policy and the annotation works in my test cluter.